larpandoracontent/LArCheating/CheatingNeutrinoIdTool.cc

```diff
@@ -19,8 +19,10 @@ using namespace pandora;
 namespace lar_content
 {
 
-void CheatingNeutrinoIdTool::SelectOutputPfos(const SliceHypotheses &nuSliceHypotheses, const SliceHypotheses &crSliceHypotheses, PfoList &selectedPfos)
+void CheatingNeutrinoIdTool::SelectOutputPfos(const MasterAlgorithm *const pAlgorithm, const SliceHypotheses &nuSliceHypotheses, const SliceHypotheses &crSliceHypotheses, PfoList &selectedPfos)
 {
+    (void) pAlgorithm;
```

Comment:
Just remove (comment out better) algorithm variable, rather than add a null line to implementation. (And similar, below.)